calibration.py

```diff
@@ -0,0 +1,55 @@
+__author__ = 'Patrick B. Grinaway'
+
+import openeye.oechem as oechem
+import gaff2xml
+import simtk.openmm as openmm
+import simtk.openmm.app as app
+import simtk.unit as units
+import joblib
+
+class Calibrator(object):
+
+    """
+    Given a ligand system, generate new sets of charges
+    representing fluorination states, as well as biases for the expanded ensemble simulation in implicit solvent.
+    """
+
+    def __init__(self, ligand_oemol, ligand_prmtop, ligand_inpcrd, openmm_platform='CPU'):
```

Comment:
Do we need to generalize this besides ligands, or does the `Calibrator` class only apply to ligands?